org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/JDTUtils.java

```diff
@@ -196,6 +198,14 @@ public static ICompilationUnit resolveCompilationUnit(URI uri) {
 
 		IFile resource = (IFile) findResource(uri, ResourcesPlugin.getWorkspace().getRoot()::findFilesForLocationURI);
 		if(resource != null) {
+			ICompilationUnit[] workingCopies = JavaModelManager.getJavaModelManager().getWorkingCopies(DefaultWorkingCopyOwner.PRIMARY, false);

```

Comment:
This can be replaced by `JavaCore.getWorkingCopies(null)` ?

Reply:
Fixed.